rebasehelper/application.py

```diff
@@ -708,7 +717,7 @@ def run(self):
                     Application.check_build_requires(self.spec_file)
                 # Build packages
                 try:
-                    build = self.build_packages()
+                    build = self.build_packages
```

Comment:
Why did you delete brackets? Is it working? I did not know, brackets are not needed? Is this python 3 specific?